dspace-api/src/main/java/org/dspace/app/util/AdditionalRelationshipScript.java

```diff
@@ -0,0 +1,388 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.util;
+
+import java.sql.SQLException;
+
+import org.apache.commons.cli.ParseException;
+import org.dspace.authorize.AuthorizeException;
+import org.dspace.content.Item;
+import org.dspace.content.Relationship;
+import org.dspace.content.RelationshipType;
+import org.dspace.content.factory.ContentServiceFactory;
+import org.dspace.content.service.EntityTypeService;
+import org.dspace.content.service.ItemService;
+import org.dspace.content.service.RelationshipService;
+import org.dspace.content.service.RelationshipTypeService;
+import org.dspace.core.Context;
+
+/**
+ * This script is used to populate the database with specific objects (see UUIDs) with a set of
+ * relationships so that various functionalities can be tested properly.
```

Comment:
The description of this script sounds like it should be a Unit Test script? 

I'm assuming it's only meant for generating test data...in which case, we *really* either need to ensure this gets removed (eventually) or we need it to be changed into a Unit Test resource.  We shouldn't have a "demo" script in the main codebase, obviously ;)

Reply:
The goal for this class was to create an easy test setup with some initial items, and to remove it or move it to a Unit Test later-on
But upon further review, it seems to also be unable to create that test setup, so I think we can better remove it and identify another solution to make it easier to populate a test repository for verifying the functionality

Reply:
This should be resolved using commit b37d0e216718316ba202f34556f53344cd0346b8